l10n_ch_payment_slip/__manifest__.py

```diff
@@ -1,32 +1,28 @@
-# -*- coding: utf-8 -*-
-# © 2012-2016 Camptocamp SA
+# Copyright 2012-2016 Camptocamp SA
```

Comment:
2012-2018, same for other py files

Reply:
When significant changes were made in py files I updated it already (was in 2017 BTW)